game-state-service

```diff
@@ -24,12 +27,25 @@ class GameState(object):
         try:
             parser.load_from_file(self._FILE_PATH)
             root = parser.get_root()
-            assert root is not None
+            if root is None:
+                raise InvalidStateError()
             self._state = root.get_object()
-            assert self._state is not None
-        except (GLib.Error, AssertionError):
-            traceback.print_exc()
+            if self._state is None:
+                raise InvalidStateError()
+            reset = False
+        except GLib.Error as e:
+            if GLib.quark_from_string(e.domain) == GLib.file_error_quark():
+                print('Can\'t read state from file %s.' % self._FILE_PATH)
```

Comment:
Consider using `logging` (https://docs.python.org/3.5/library/logging.html) for these messages. Fabian has a nice example in hack-sound-server.

Reply:
sure (I was planing to add proper logging support in T24185).